src/test/java/com/bytezone/dm3270/TerminalClientTest.java

```diff
@@ -355,4 +367,214 @@ private void setupLoginWithSscpLuData() throws Exception {
     connectClient();
   }
 
+  //TODO .getField before connect to it shouldThrowSomethingWhenGetFieldBeforeConnect
+  @Test
+  public void shouldGetCorrectFieldsWhenGetFields() throws Exception {
+    awaitKeyboardUnlock();
+    sendUserFieldByCoord();
+    awaitKeyboardUnlock();
+    List<Field> list2 = client.getFields();
+    List<Field> list = expectedFields();
+    assertEquals(list2, list);
```

Comment:
the order should be inverted, since assertEquals first parameter is defined as the expected one and the second parameter is the actual one. Additionally you might just inline list and list2 variables since expressions to get them are so simple.